trigger-openqa_in_openqa

```diff
@@ -46,14 +47,18 @@ trigger() {
     if [ "$target_host" = "openqa.opensuse.org" ]; then
         ARGS="OPENQA_HOST=http://openqa.opensuse.org"
     fi
+    # in full run use only openqa_install+publish test
+    if [ "$full_run" ]; then
+        ARGS+=" TEST=\'openqa_install+publish\' FULL_OPENSUSE_TEST=1"
+    fi
 
     ${client_prefix} "${openqa_cli}" \
         api --host "${target_host_proto}://${target_host}" -X POST isos \
         VERSION=Tumbleweed \
         DISTRI=openQA FLAVOR=dev BUILD="${build}" \
         ARCH="${arch}" BACKEND=qemu WORKER_CLASS=qemu_x86_64 \
         HDD_1="$qcow" \
-        ${ARGS} \
+        "${ARGS}" \
```

Comment:
I don't think quoting will work as the key-value-pairs actually need to be passed as separate arguments. Maybe it makes sense to make `ARGS` a bash array. Then the quoting would make sense.

Reply:
why?  it passes and parses the whole cli as a string

Reply:
+ without quotes is ShellCheck unhappy

Reply:
> without quotes is ShellCheck unhappy

That's true.

> why? it passes and parses the whole cli as a string

If it works then fine :-)

Reply:
How do we know it works? Do we have tests? If we do, why don't they fail since they were not modified in this PR 🤐😉